src/components/table/table.tsx

```diff
@@ -71,6 +78,18 @@ export class Table {
     @Event()
     public load: EventEmitter<TableParams>;
 
+    /**
+     * Emitted when a raw is selected
+     */
+    @Event()
+    public selectRow: EventEmitter<number>;
+
+    /**
+     * Emitted when `mode` is `local` and a new page has been set
+     */
+    @State()
+    public prevSelectedRawId: number;
```

Comment:
Right now we allow the user to select multiple rows since the `selectable` property is a `number`, but we only store a single row here. I think we need to store all the rows that are selected :thinking: 

Also, I think it should be possible to have rows pre-selected when the table is created, so this should probably be a `@Prop` instead. However, in the `limel-list` component `selected` is part of the data that is given to the component so we might want to do something similar here. I'm not sure what the best approach is :thinking: